package.json

```diff
@@ -52,7 +52,7 @@
     "ts-jest": "~29",
     "ts-loader": "9.4.1",
     "ts-node": "^10.8.1",
-    "typescript": "4.8.4",
+    "typescript": "~4",
```

Comment:
I would keep a version pinned to be sure there isn't unexpected build issue in the ci or suddently locally

Reply:
How would that happen since we have the lockfile?

Reply:
If you run any `npm i` or `npm i another_lib` it might be bumped my understanding. So yeah less chance in the CI, more chance locally. Maybe not tomorrow, not after tomorrow, but some day.